scipy/sparse/linalg/_dsolve/tests/test_linsolve.py

```diff
@@ -743,12 +784,12 @@ def test_zero_diagonal(self):
         A = np.array([[0, 0, 0], [1, 0, 0], [1, 1, 0]], dtype=np.float64)
         b = np.array([1., 2., 3.])
         with suppress_warnings() as sup:
-            sup.filter(SparseEfficiencyWarning, "CSR matrix format is")
+            sup.filter(SparseEfficiencyWarning, "CSC or CSR matrix format is")
             spsolve_triangular(A, b, unit_diagonal=True)
 
     def test_singular(self):
         n = 5
-        A = csr_matrix((n, n))
+        A = csc_matrix((n, n))
```

Comment:
Change from CSR to CSC

Reply:
Thanks, made a parametric test with both formats in [e97868d](https://github.com/scipy/scipy/pull/17924/commits/e97868dfca28071066e0a38b3e9e41849eb4d713)